universal/security/uefisecureboot.md

```diff
@@ -0,0 +1,257 @@
+# UEFI Secure Boot
+
+* Based off of CodeRush's [Taming UEFI SecureBoot](https://habr.com/en/post/273497/)
+
+
+To start setting up UEFI Secure Boot on your machine, we'll first need to ensure a few things:
+
+
+* Apple Secure Boot Page has been followed correctly
+  * This includes DmgLoading and SecureBootModel, ApECID is optional however we strongly encourage you to set this up as well
+  * Reminder DmgLoading must be set to either `Signed` or `Disabled`
+* You've setup Vaulting
+  * Note Vaulting and FileVault are different, FieVault is not require for UEFI Secure Boot however is still strongly encouraged
```

Comment:
```suggestion
  * Note Vaulting and FileVault are different, FileVault is not require for UEFI Secure Boot however is still strongly encouraged
```

Typo fix

Reply:
There is a grammar issue here: require to required